OrbitGl/Track.h

```diff
@@ -57,8 +57,8 @@ class Track : public Pickable, public std::enable_shared_from_this<Track> {
   void SetVisible(bool value) { visible_ = value; }
 
   [[nodiscard]] uint32_t GetNumTimers() const { return num_timers_; }
-  [[nodiscard]] uint64_t GetMinTime() const { return min_time_; }
-  [[nodiscard]] uint64_t GetMaxTime() const { return max_time_; }
+  [[nodiscard]] virtual uint64_t GetMinTime() const { return min_time_; }
```

Comment:
This is only done do have it overridden in `EventTrack`, right? I think this is really confusing, as far as I see the `min_time_` / `max_time_` are only set to meaningful values for the `ThreadTrack`, in which case it is actually computed from it's `EventTrack` child, which in turn passes it through from the `capture_data_`...

But IMHO this is a symptom of `EventTrack` inheriting `Track`, which is something we should get rid of. Not in this PR, since it's quite big anyways, but I'd love to see this refactored later and then have this change actually undone.

Reply:
Yes I agree, this should be cleaned up. I still think it's useful for some types of Tracks, but not all. The GetNumTimers has no business in the main Track class for example, it should be in the TimerTrack.